ansible/roles/test/files/tools/loganalyzer/loganalyzer.py

```diff
@@ -360,19 +363,19 @@ def analyze_file(self, log_file_path, match_messages_regex, ignore_messages_rege
                     in_analysis_range = False
                     break
 
-            if in_analysis_range or is_sairedis_rec:
+            if in_analysis_range:
                 # Skip long logs in sairedis recording since most likely they are bulk set operations for non-default routes
                 # without much insight while they are time consuming to analyze
-                if is_sairedis_rec and len(rev_line) > 1000:
+                if not check_marker and len(rev_line) > 1000:
```

Comment:
`in_analysis_range` is already defined as `not check_marker`, so check `not check_marker` again is redundant?

Reply:
`in_analysis_range` is defined as `not check_marker` as an initial value. It is possible that `in_analysis_range` and `check_marker` are both True. https://github.com/Azure/sonic-mgmt/blob/9585036c8ace56bf4dba0f6444c854f838d5a8a3/ansible/roles/test/files/tools/loganalyzer/loganalyzer.py#L357